sandbox/chris-gan/implicit_style_fixed_samples/style.py

```diff
@@ -68,10 +68,10 @@ def __init__(self, z_dim, width, num_layers):
             [nn.Linear(width, width) for _ in range(num_layers)])
 
     def forward(self, z):
-        a = F.elu(self.fc(z))
+        a = F.leaky_relu(self.fc(z), 0.2)
 
         for layer in self.layers:
-            a = F.elu(a)
+            a = F.leaky_relu(layer(a), 0.2)
```

Comment:
Womp, I forgot to forward through the layer in this loop, so things were just being repeatedly activated, and it wasn't actually an mlp.

Reply:
Big womp